README.md

```diff
@@ -137,6 +138,7 @@ public class PersonComponent implements IsElement<HTMLDivElement>, Editor<Person
 
     private DominoElement<HTMLDivElement> root = DominoElement.of(div());
 
+    //Note:  Do not create getter/setters for these fields - it will cause a compile issue such as "java.lang.IllegalStateException: generation aborted! No getter exists for >>getId<< -> [Help 1]"
```

Comment:
probably we should have a bug report for this?

Reply:
Bug Report Submitted
https://github.com/gwtproject/gwt-editor/issues/26

Reply:
I don't think, that there is a problem with the example. Please, take a look at my comment inside the issue.